plugins/modules/system/parted.py

```diff
@@ -557,12 +557,12 @@ def parted(script, device, align):
         align_option = ''
 
     if script and not module.check_mode:
-        command = "%s -s -m %s %s -- %s" % (parted_exec, align_option, device, script)
+        command = [parted_exec, '-s', '-m', align_option, device, '--', script]
```

Comment:
`script` currently sometimes has double quotes, see for example https://github.com/ansible-collections/community.general/blob/47bd06eb94065b67a968dfd23b0de5f06473b2ce/plugins/modules/system/parted.py#L746.

Reply:
yep, fixed